python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -413,6 +413,28 @@ def ElementwiseLeakyReluModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ElementwisePreluModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+        ([-1, -1], torch.float32, True),
```

Comment:
If `weight` argument is rank-1, this should be `[-1]`. See https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#export-and-annotate_args-decorators for an explanation on what the arguments here mean.

Reply:
Understood, thank for reference.